Build failure: sdrangel

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

using latest nixpkgs-unstable, put sdrangel inside system configuration.
For some reasons the last 8 builds on hydra failed for various/inconsistent dependencies issues (eg. [this](https://hydra.nixos.org/build/315350028)), and the previous 2 builds are marked as "build failed" but do not have logs explaining why (eg. [this](https://hydra.nixos.org/build/314939654)).

### Can Hydra reproduce this build failure?

No, Hydra cannot reproduce this build failure.

### Link to Hydra build job

_No response_

### Relevant log output

```console
@nix { "action": "setPhase", "phase": "configurePhase" }
cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMA
KE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/dqwkz6fl25m1vflci5sdmva4r997c35p-sdrangel-7.22.9/share/locale -DCMAK
E_INSTALL_LIBEXECDIR=/nix/store/dqwkz6fl25m1vflci5sdmva4r997c35p-sdrangel-7.22.9/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/dqwkz6fl25m1vflci5sdmva4
r997c35p-sdrangel-7.22.9/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/dqwkz6fl25m1vflci5sdmva4r997c35p-sdrangel-7.22.9/share/doc/sdrangel -DCMAKE_INSTALL_
INFODIR=/nix/store/dqwkz6fl25m1vflci5sdmva4r997c35p-sdrangel-7.22.9/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/dqwkz6fl25m1vflci5sdmva4r997c35p-s
drangel-7.22.9/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/dqwkz6fl25m1vflci5sdmva4r997c35p-sdrangel-7.22.9/include -DCMAKE_INSTALL_SBINDIR=/ni
x/store/dqwkz6fl25m1vflci5sdmva4r997c35p-sdrangel-7.22.9/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/dqwkz6fl25m1vflci5sdmva4r997c35p-sdrangel-7.22.9/bi
n -DCMAKE_INSTALL_NAME_DIR=/nix/store/dqwkz6fl25m1vflci5sdmva4r997c35p-sdrangel-7.22.9/lib -DCMAKE_STRIP=/nix/store/vr15iyyykg9zai6fpgvhcgyw7gckl78
w-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/vr15iyyykg9zai6fpgvhcgyw7gckl78w-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/vr15i
yyykg9zai6fpgvhcgyw7gckl78w-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/dqwkz6fl25m
1vflci5sdmva4r997c35p-sdrangel-7.22.9 -DAPT_DIR=/nix/store/2gkgyhd404zp34kss69yka121sn9gwf9-aptdec-1.8.0 -DDAB_DIR=/nix/store/mzb1k9p4q90w3fpz4g92b
bshwf83nidn-dab_lib-unstable-2023-03-02 -DSGP4_DIR=/nix/store/nnl8wyn2j3071ajqhdzxa8i4xghnlidv-sgp4-unstable-2022-11-13 -DSOAPYSDR_DIR=/nix/store/i
zfjkf97bk7y6sydnxyn6225lamnvg9f-soapysdr-0.8.1-unstable-2025-10-05-03 -Wno-dev -DENABLE_QT6=ON -DQT_OPTIONAL_TOOLS_PATH=/nix/store/q0v782wajb13g2xh
6vypazvcmwlpc1qg-qttools-6.10.1 -DQT_OPTIONAL_TOOLS_PATH=/nix/store/q0v782wajb13g2xh6vypazvcmwlpc1qg-qttools-6.10.1 -DPROTOC_EXE=/nix/store/8lql16s
1xfd5rmi73aa80ivkg09py9n5-protobuf-32.1/bin/protoc -DProtobuf_PROTOC_EXE=/nix/store/8lql16s1xfd5rmi73aa80ivkg09py9n5-protobuf-32.1/bin/protoc -DPro
tobuf_PROTOC_EXECUTABLE=/nix/store/8lql16s1xfd5rmi73aa80ivkg09py9n5-protobuf-32.1/bin/protoc
-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/vr15iyyykg9zai6fpgvhcgyw7gckl78w-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/vr15iyyykg9zai6fpgvhcgyw7gckl78w-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find Git (missing: GIT_EXECUTABLE)
-- SDRangel Version: 7.22.9
-- Build type set to: Release
-- Compiling for 24 bit Rx DSP chain
-- Looking for __x86_64__
-- Looking for __x86_64__ - found
-- Target architecture: x86_64-native
-- Performing Test FLAG_SUPPORTED
-- Performing Test FLAG_SUPPORTED - Success
-- Looking for __ARM_NEON
-- Looking for __ARM_NEON - not found
-- Looking for __AVX512F__
-- Looking for __AVX512F__ - not found
-- Looking for __AVX2__
-- Looking for __AVX2__ - not found
-- Looking for __AVX__
-- Looking for __AVX__ - not found
-- Looking for __SSE4_2__
-- Looking for __SSE4_2__ - not found
-- Looking for __SSE4_1__
-- Looking for __SSE4_1__ - not found
-- Looking for __SSSE3__
-- Looking for __SSSE3__ - not found
-- Looking for __SSE3__
-- Looking for __SSE3__ - not found
-- Looking for __SSE2__
-- Looking for __SSE2__ - found
-- Interprocedural optimization enabled
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found OpenGL: /nix/store/mm4v6w3wj8wi5qc6rwdrb2rkqs08whp9-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/j0jc9819vx3gdky3jfxp9fwza54xk184-vulkan-headers-1.4.328.0/include
-- Found Cups: /nix/store/yp9c76fj59a9lcn4afp4rylrhjzqk129-cups-2.4.14-lib/lib/libcups.so (found version "2.4.14")
-- Found PkgConfig: /nix/store/8vdiwpbh0g4avsd6x5v4s0di32vcl3dp-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Found Boost: /nix/store/k0m9kall5vjhn60sqs8q9qfn31i4f31f-boost-1.87.0-dev/lib/cmake/Boost-1.87.0/BoostConfig.cmake (found version "1.87.0")
-- Checking for module 'fftw3f >= 3.0'
--   Found fftw3f, version 3.3.10
-- Found FFTW3F: /nix/store/kkmg40b5yz3kn0qaz6rcsxgq2hr6chmm-fftw-single-3.3.10/lib/libfftw3f.so
-- Found FFTW3F: /nix/store/yimshs6yhbhclabqkhjijpdfj2isxc9r-fftw-single-3.3.10-dev/include, /nix/store/kkmg40b5yz3kn0qaz6rcsxgq2hr6chmm-fftw-single-3.3.10/lib/libfftw3f.so, /nix/store/kkmg40b5yz3kn0qaz6rcsxgq2hr6chmm-fftw-single-3.3.10/lib/libfftw3f_threads.so
-- Checking for module 'libusb-1.0'
--   Found libusb-1.0, version 1.0.29
-- Found libusb-1.0: /nix/store/2wjb7lnk2jhn0m3n1bgmk7p83aznjdm7-libusb-1.0.29-dev/include/libusb-1.0, /nix/store/00s3yaabi3v8y64qq5jwgg5g4dlnmcpq-libusb-1.0.29/lib/libusb-1.0.so
-- Found OpenCV: / (found version "4.12.0") found components: core highgui imgproc imgcodecs videoio
-- Checking for module 'libsigmf'
--   No package 'libsigmf' found
-- libsigmf not found.
-- Found ZLIB: /nix/store/l7xwm1f6f3zj2x8jwdbi8gdyfbx07sh7-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Checking for module 'faad2'
--   Found faad2, version 2.11.2
-- FAAD LIBRARY /nix/store/sfnswcr5y8v8kk6f0lp1b7gzpdq4rx2q-faad2-2.11.2/lib/libfaad.so
-- FAAD INCLUDE DIRS /nix/store/3hbl3y1iymrjr9wsprplafrp87mywc2x-faad2-2.11.2-dev/include
-- Found FAAD: /nix/store/sfnswcr5y8v8kk6f0lp1b7gzpdq4rx2q-faad2-2.11.2/lib/libfaad.so
-- Checking for module 'codec2'
--   Found codec2, version 1.2.0
-- Found Codec2: /nix/store/bl9db0656yzk7lcg1l63rsh7vlnnjlnh-codec2-1.2.0-dev/include, /nix/store/ik3yqwj5npgifzpkf11b5f8zghzh52zj-codec2-1.2.0-lib/lib/libcodec2.so
-- Found CODEC2: /nix/store/ik3yqwj5npgifzpkf11b5f8zghzh52zj-codec2-1.2.0-lib/lib/libcodec2.so
-- Checking for module 'libcm256cc'
--   Found libcm256cc, version 1.1.0
-- Found CM256cc: /nix/store/4hv42mp032s7ymai4fl3zyhljm340rwy-cm256cc-1.1.2/include, /nix/store/4hv42mp032s7ymai4fl3zyhljm340rwy-cm256cc-1.1.2/lib/libcm256cc.so
-- Found CM256CC: /nix/store/4hv42mp032s7ymai4fl3zyhljm340rwy-cm256cc-1.1.2/lib/libcm256cc.so
-- Found LibMbe: /nix/store/j1afpr74jfphydzzivhxzykvj0sya90f-mbelib-1.3.0/include,
-- Checking for module 'libdsdcc'
--   Found libdsdcc, version 1.9.0
-- Found libdsdcc: /nix/store/0aqs98ajqdavnsd6grjb3p7lz235hsjh-dsdcc-1.9.6/include, /nix/store/0aqs98ajqdavnsd6grjb3p7lz235hsjh-dsdcc-1.9.6/lib/libdsdcc.so
-- Checking for module 'sgp4'
--   No package 'sgp4' found
-- SGP4 LIBRARIES /nix/store/nnl8wyn2j3071ajqhdzxa8i4xghnlidv-sgp4-unstable-2022-11-13/lib/libsgp4s.so
-- SGP4 INCLUDE DIRS /nix/store/nnl8wyn2j3071ajqhdzxa8i4xghnlidv-sgp4-unstable-2022-11-13/include/libsgp4
-- Found SGP4: /nix/store/nnl8wyn2j3071ajqhdzxa8i4xghnlidv-sgp4-unstable-2022-11-13/lib/libsgp4s.so
-- Checking for module 'apt'
--   No package 'apt' found
-- APT LIBRARIES /nix/store/2gkgyhd404zp34kss69yka121sn9gwf9-aptdec-1.8.0/lib/libapt.so
-- APT INCLUDE DIRS /nix/store/2gkgyhd404zp34kss69yka121sn9gwf9-aptdec-1.8.0/include/apt
-- Found APT: /nix/store/2gkgyhd404zp34kss69yka121sn9gwf9-aptdec-1.8.0/lib/libapt.so
-- Checking for module 'libdab'
--   No package 'libdab' found
-- Found libdab: /nix/store/mzb1k9p4q90w3fpz4g92bbshwf83nidn-dab_lib-unstable-2023-03-02/include/dab_lib, /nix/store/mzb1k9p4q90w3fpz4g92bbshwf83nidn-dab_lib-unstable-2023-03-02/lib/libdab_lib.so
-- Found HIDAPI: /nix/store/s1jxcydakf1862wgcny3ac02hwcs9dwc-hidapi-0.15.0/include/hidapi  found components: any
-- Checking for module 'libavcodec'
--   Found libavcodec, version 62.11.100
-- Checking for module 'libavformat'
--   Found libavformat, version 62.3.100
-- Checking for module 'libavdevice'
--   Found libavdevice, version 62.1.100
-- Checking for module 'libavutil'
--   Found libavutil, version 60.8.100
-- Checking for module 'libavfilter'
--   Found libavfilter, version 11.4.100
-- Checking for module 'libswscale'
--   Found libswscale, version 9.1.100
-- Checking for module 'libpostproc'
--   No package 'libpostproc' found
-- Checking for module 'libswresample'
--   Found libswresample, version 6.1.100
-- Found FFmpeg: /nix/store/smns0ny2jjfs4vm2y4zv0bdanwx00fdy-ffmpeg-8.0-lib/lib/libavcodec.so;/nix/store/smns0ny2jjfs4vm2y4zv0bdanwx00fdy-ffmpeg-8.0-lib/lib/libavformat.so;/nix/store/smns0ny2jjfs4vm2y4zv0bdanwx00fdy-ffmpeg-8.0-lib/lib/libavutil.so;/nix/store/smns0ny2jjfs4vm2y4zv0bdanwx00fdy-ffmpeg-8.0-lib/lib/libswscale.so
-- Checking for module 'libggmorse'
--   No package 'libggmorse' found
-- GGMorse not found
-- Could NOT find GGMORSE (missing: GGMORSE_LIBRARIES GGMORSE_INCLUDE_DIR)
-- Checking for module 'libairspy'
--   Found libairspy, version 1.0
-- Found libairspy: /nix/store/6lmvvcnws3h4a7zmd9kfhzxfvdddvh5v-airspy-1.0.10/include, /nix/store/6lmvvcnws3h4a7zmd9kfhzxfvdddvh5v-airspy-1.0.10/lib/libairspy.so
-- Checking for module 'libairspyhf'
--   Found libairspyhf, version 1.8
-- Found libairspyhf: /nix/store/apr1ldd761c5i3r1avv37aajchck7bv1-airspyhf-1.6.8-unstable-2025-07-12/include, /nix/store/apr1ldd761c5i3r1avv37aajchck7bv1-airspyhf-1.6.8-unstable-2025-07-12/lib/libairspyhf.so
-- Checking for module 'libbladeRF>=2.0'
--   Found libbladeRF, version 2.6.0-git-unknown
-- Found libbladerf: /nix/store/k3fksj9zr98smfzfrplzcr797pzlnqlw-libbladeRF-2025.10/include, /nix/store/k3fksj9zr98smfzfrplzcr797pzlnqlw-libbladeRF-2025.10/lib/libbladeRF.so
-- Checking for module 'libhackrf'
--   Found libhackrf, version 0.9
-- Found libhackrf: /nix/store/b8bdp1zfirfwykj4fkq89rwa6n482fh9-hackrf-2024.02.1/include, /nix/store/b8bdp1zfirfwykj4fkq89rwa6n482fh9-hackrf-2024.02.1/lib/libhackrf.so
-- Found Lime Suite: /nix/store/yb91n23dk8nwzxfalxi5wpr03wbhp9ai-limesuite-23.11.0/include, /nix/store/yb91n23dk8nwzxfalxi5wpr03wbhp9ai-limesuite-23.11.0/lib/libLimeSuite.so
-- Checking for module 'libiio>=0.7'
--   Found libiio, version 0.26
-- Found libiio: /nix/store/3njx8dxqf5dcpr0lcsp9wjhp9lqrhccs-libiio-0.26-dev/include, /nix/store/2lf4yixb9qp3k89mbal581jxbvag94a0-libiio-0.26-lib/lib/libiio.so
-- Checking for module 'libmirisdr'
--   No package 'libmirisdr' found
-- libmirisdr not found.
-- Checking for module 'libperseus'
--   No package 'libperseus' found
-- libperseus not found.
-- Checking for module 'librtlsdr'
--   Found librtlsdr, version 0.6.5
-- Found librtlsdr: /nix/store/wala66r1srb4hy9ds0fhyybh9dskfflc-rtl-sdr-blog-1.3.5/include, /nix/store/wala66r1srb4hy9ds0fhyybh9dskfflc-rtl-sdr-blog-1.3.5/lib/librtlsdr.so
-- Checking for one of the modules 'SoapySDR>=0.4.0'
-- Found SoapySDR: version 0.8-3, /nix/store/izfjkf97bk7y6sydnxyn6225lamnvg9f-soapysdr-0.8.1-unstable-2025-10-05-03, /nix/store/izfjkf97bk7y6sydnxyn6225lamnvg9f-soapysdr-0.8.1-unstable-2025-10-05-03/include, /nix/store/izfjkf97bk7y6sydnxyn6225lamnvg9f-soapysdr-0.8.1-unstable-2025-10-05-03/lib/libSoapySDR.so
-- Checking for module 'uhd'
--   Found uhd, version 4.9.0.0-0-unknown
-- UHD LIBRARIES /nix/store/13v83pivf8x6hayf75syizqj8hwmnmjm-uhd-4.9.0.0/lib/libuhd.so
-- UHD INCLUDE DIRS /nix/store/7a92ca542x8xdi7n97fkna5f277bqinx-uhd-4.9.0.0-dev/include
-- Found UHD: /nix/store/13v83pivf8x6hayf75syizqj8hwmnmjm-uhd-4.9.0.0/lib/libuhd.so
-- SDRPLAY LIBRARIES SDRPLAY_LIBRARIES-NOTFOUND
-- SDRPLAY INCLUDE DIRS SDRPLAY_INCLUDE_DIR-NOTFOUND
-- Could NOT find SDRPLAY (missing: SDRPLAY_LIBRARIES SDRPLAY_INCLUDE_DIR)
-- Checking for module 'libxtrx'
--   No package 'libxtrx' found
-- libxtrx not found.
-- Found Opus: /nix/store/szf4nh1700qn38hqnra2pgj1i35pw7ks-libopus-1.5.2-dev/include
-- Could not find nvcc, please set CUDAToolkit_ROOT.
-- Not building remotesink (ENABLE_CHANNELRX_REMOTESINK=ON CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building sigmffilesink (ENABLE_CHANNELRX_SIGMFFILESINK=ON LIBSIGMF_FOUND=FALSE)
-- Include demoddatv
-- Not building remotesource (ENABLE_CHANNELTX_REMOTESOURCE=ON CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building ambe (ENABLE_FEATURE_AMBE=ON LIBSERIALDV_FOUND=FALSE)
-- Not building morsedecoder (ENABLE_FEATURE_MORSEDECODER=ON GGMODSE_FOUND=FALSE)
-- Not building xtrxmimo (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Checking for module 'hamlib'
--   No package 'hamlib' found
-- Hamlib not found
-- Could NOT find HAMLIB (missing: hamlib_LIBRARIES hamlib_INCLUDE_DIRS)
-- Not building audiocatsiso (hamlib_FOUND=FALSE)
-- Not building remoteinput (CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building sdrplay (ENABLE_MIRISDR=ON LIBMIRISDR_FOUND=FALSE)
-- Not building sdrplayv3 (ENABLE_SDRPLAY=ON SDRPLAY_FOUND=FALSE)
-- Not building perseus (ENABLE_PERSEUS=ON LIBPERSEUS_FOUND=FALSE RX_SAMPLE_24BIT=ON)
-- Not building xtrxinput (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Not building sigmffileinput (LIBSIGMF_FOUND=FALSE)
-- Not building remoteoutput (CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building xtrxoutput (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Not building remotesink (ENABLE_CHANNELRX_REMOTESINK=ON CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building sigmffilesink (ENABLE_CHANNELRX_SIGMFFILESINK=ON LIBSIGMF_FOUND=FALSE)
-- Include demoddatv
-- Not building remotesource (ENABLE_CHANNELTX_REMOTESOURCE=ON CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building jogdialcontroller (ENABLE_FEATURE_JOGDIALCONTROLLER=ON SERVER_MODE=ON)
-- Not building ambe (ENABLE_FEATURE_AMBE=ON LIBSERIALDV_FOUND=FALSE)
-- Not building morsedecoder (ENABLE_FEATURE_MORSEDECODER=ON GGMODSE_FOUND=FALSE)
-- Not building xtrxmimo (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Checking for module 'hamlib'
--   No package 'hamlib' found
-- Hamlib not found
-- Could NOT find HAMLIB (missing: hamlib_LIBRARIES hamlib_INCLUDE_DIRS)
-- Not building audiocatsiso (hamlib_FOUND=FALSE)
-- Not building remoteinput (CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building sdrplay (ENABLE_MIRISDR=ON LIBMIRISDR_FOUND=FALSE)
-- Not building sdrplayv3 (ENABLE_SDRPLAY=ON SDRPLAY_FOUND=FALSE)
-- Not building perseus (ENABLE_PERSEUS=ON LIBPERSEUS_FOUND=FALSE RX_SAMPLE_24BIT=ON)
-- Not building xtrxinput (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Not building sigmffileinput (LIBSIGMF_FOUND=FALSE)
-- Not building remoteoutput (CM256CC_FOUND=TRUE HAS_SSE3= HAS_NEON=)
-- Not building xtrxoutput (ENABLE_XTRX=ON LIBXTRX_FOUND=FALSE)
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
-- Doxygen not found, developer documentation can not be generated
-- Asking qmake for QT_PLUGINS_DIR and got /nix/store/zp6r9bxds1hldvkx1vbrk0d1ady17zhh-qtbase-6.10.1/lib/qt-6/plugins
-- Asking qmake for QT_IMPORTS_DIR and got **Unknown**
-- Asking qmake for QT_QML_DIR and got /nix/store/zp6r9bxds1hldvkx1vbrk0d1ady17zhh-qtbase-6.10.1/lib/qt-6/qml
-- Asking qmake for QT_LIBS_DIR and got /nix/store/zp6r9bxds1hldvkx1vbrk0d1ady17zhh-qtbase-6.10.1/lib
-- CPack generators:
-- Configuring done (7.6s)
CMake Error at sdrbase/CMakeLists.txt:710 (target_link_libraries):
  Target "sdrbase" links to:

    Qt::CorePrivate

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



-- Generating done (2.2s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    PROTOC_EXE
    Protobuf_PROTOC_EXE
    Protobuf_PROTOC_EXECUTABLE


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

### Additional context

nixpkgs ref: 2d293cbfa5a793b4c50d17c05ef9e385b90edf6c

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.17.8, NixOS, 25.11 (Xantusia), 25.11.20251122.050e09e`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/lrp7nzx3k8myna6hr9vz0liqmqxmkysk-source`

### Notify maintainers

@aciceri @alkeryn @Tungsten842

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/sd/sdrangel/package.nix
+++ b/pkgs/by-name/sd/sdrangel/package.nix
@@ -12,6 +12,7 @@
   dsdcc,
   faad2,
   fetchFromGitHub,
+  fetchpatch,
   fftwFloat,
   flac,
   glew,
@@ -50,6 +51,14 @@ stdenv.mkDerivation (finalAttrs: {
     tag = "v${finalAttrs.version}";
     hash = "sha256-VYSM9ldzx/8tWKQb++qGROSXdeEXIDhGqnnHUmkW4+k=";
   };
+
+  patches = [
+    # Fix build with Qt 6.10, remove when the commit reaches a release
+    (fetchpatch {
+      url = "https://github.com/f4exb/sdrangel/commit/fd6a8d51f8c39fd31b4e864f528bf1921ebd4260.patch";
+      hash = "sha256-S8LQbCTEgyEt1wByDsDMqqyQjK5HALtvUIODgQ1skSA=";
+    })
+  ];
 
   nativeBuildInputs = [
     cmake

